src/app/engage-with-us/page.tsx

```diff
@@ -13,10 +13,6 @@ import Hero from '../components/Hero/Hero';
 export default function EngageWithUs() {
```

Comment:
We'll need to remove the `import Hero from '../components/Hero/Hero';` line from this file. That's what is causing the eslint error.

Reply:
done